MediaBrowser.Providers/MediaInfo/AudioFileProber.cs

```diff
@@ -276,7 +281,14 @@ private async Task FetchDataFromTags(Audio audio, Model.MediaInfo.MediaInfo medi
 
                     if (!audio.PremiereDate.HasValue)
                     {
-                        audio.PremiereDate = new DateTime(year, 01, 01);
+                        try
+                        {
+                            audio.PremiereDate = new DateTime(year, 01, 01);
+                        }
+                        catch (ArgumentOutOfRangeException ex)
+                        {
+                            _logger.LogError(ex, "Error getting YEAR value for audio file.");
```

Comment:
I am not familiar with our logging mechanism. But does this also end up showing somewhere what file generated this error, in the log? 

Also, I am unsure about the term "Getting". Perhaps "Parsing" (i.e. Error parsing year from audio file) is a better alternative? But that is just a nitpick from me and maybe I am wrong.

Reply:
Good ideas, I updated the message!